### ISSUE TITLE
Feature request: Change to one of the symbols to imiprove visual distinction

### ISSUE THREAD

**Issue by BarryTice** [Softmotions/einstein-mobile#5]
Greetings.

In the shapes row in the field of symbols, the game includes both a pentagon (&#11040;) and an inverted pentagon. When I do not have my glasses on, it is easy for me to mistake one of these for the other at small sizes.

Would it be possible to replace the upright pentagon with a pentagram/star (&#9734;) instead? This would provide improved visual distinction between the two shapes.

Thank you for your consideration.

**Comment by Noah-Count** [comment]
I agree that this would be helpful. I have made edits to the gif files for both the hint and item graphics in all five sizes. I made the up-pointing pentagon a pentagram and then filled both it and the up-pointing triangle. this makes both of them much more clearly distinct from the ones pointing down.

I assume you guys accept pull requests?

**Comment by Byrbl** [comment]
> I agree that this would be helpful. I have made edits to the gif files for both the hint and item graphics in all five sizes. I made the up-pointing pentagon a pentagram and then filled both it and the up-pointing triangle. this makes both of them much more clearly distinct from the ones pointing down.
> 
> I assume you guys accept pull requests?

Hi. Thanks for the idea. Yes, we accept pull requests, but due to the new rules for publishing applications in the Android store, in order to release a new version, we will need some other improvements, which is not clear when it can be done. In any case, it will be interesting to look at your version of the icons.
